src/flotilla/scheduler/cloudformation.py

```diff
@@ -286,7 +286,7 @@ def service(self, region, service, vpc_outputs, stack):
                     'InstancePort': port,
                     'LoadBalancerPort': port,
                     'Protocol': protocol,
-                    'InstanceProtocol': "HTTP"
+                    'InstanceProtocol': protocol
```

Comment:
I was thinking TCP->TCP when I implemented it (fiddling with Redis as a small/light container).
I reverted to HTTP soon enough, so this is still a "_may_ fix stuff but shouldn't break anything" change.